src/remote-config/remote-config-api-client.ts

```diff
@@ -81,7 +71,7 @@ interface RemoteConfigCondition {
   tagColor?: RemoteConfigConditionDisplayColor;
 }
 
-export interface RemoteConfigContent {
+export interface RemoteConfigTemplateContent {
```

Comment:
If we turn `RemoteConfigTemplate` into a pure JS object, then we can get rid of this type.

```
export interface RemoteConfigTemplate {

}

// Not exported -- For our internal use only
class RemoteConfigTemplateImpl implements RemoteConfigTemplate {
  // class with validation
}

getTemplate() {
  return new RemoteConfigTemplateImpl(resp.data);
}
```